Objects/typeobject.c

```diff
@@ -1359,6 +1358,19 @@ _PyType_LookupByVersion(unsigned int version)
 #error "_Py_ATTR_CACHE_UNUSED must be bigger than max"
 #endif
 
+static inline unsigned int
+get_next_global_version_tag(void)
```

Comment:
Call this something like "next_global_version_tag(void)" instead of "get_next_global_version_tag".

"get_" makes it sound like a function that retrieves the next version tag without modifying it.

Reply:
done